connector/MongoDBAtlasODBC.pq

```diff
@@ -73,3 +81,64 @@ MongoDBAtlasODBC.Icons = [
     Icon16 = { Extension.Contents("MongoDBAtlasODBC16.png"), Extension.Contents("MongoDBAtlasODBC20.png"), Extension.Contents("MongoDBAtlasODBC24.png"), Extension.Contents("MongoDBAtlasODBC32.png") },
     Icon32 = { Extension.Contents("MongoDBAtlasODBC32.png"), Extension.Contents("MongoDBAtlasODBC40.png"), Extension.Contents("MongoDBAtlasODBC48.png"), Extension.Contents("MongoDBAtlasODBC64.png") }
 ];
+
+BuildOdbcConfig = () as record =>
+    let
+        Config = [
+            SqlCapabilities = [
+                FractionalSecondsScale = 3,
+                LimitClauseKind = LimitClauseKind.LimitOffset,
+                SupportsDerivedTable = true,
+                SupportsNumericLiterals = true,
+                SupportsStringLiterals = true,
+                SupportsOdbcDateLiterals = true,
+                SupportsOdbcTimeLiterals = true,
+                SupportsOdbcTimestampLiterals = true
+            ],
+            SQLGetFunctions = [
+                SQL_API_SQLBINDPARAMETER = false,
+                SQL_CONVERT_FUNCTIONS = 0x2 /* SQL_FN_CVT_CAST */
+            ],
+            SQLGetInfo = [
+                SQL_SQL_CONFORMANCE = 8 /* SQL_SC_SQL92_FULL */

```

Comment:
This conformance value is for the SELECT operations.  [Here](https://learn.microsoft.com/en-us/power-query/odbc-parameters#overriding-sqlgetinfo) it states that most drivers will report FULL compliance and override behavior using `SQLGetInfo` and `SQLGetFunctions` properties.  This will likely need further testing when we are able to test the ODBC driver.